client/core/core.go

```diff
@@ -3733,8 +3733,7 @@ func (c *Core) Orders(filter *OrderFilter) ([]*Order, error) {
 func (c *Core) coreOrderFromMetaOrder(mOrd *db.MetaOrder) (*Order, error) {
 	corder := coreOrderFromTrade(mOrd.Order, mOrd.MetaData)
 	oid := mOrd.Order.ID()
-	excludeCancels := false // maybe don't include cancel order matches?
-	matches, err := c.db.MatchesForOrder(oid, excludeCancels)
+	matches, err := c.db.MatchesForOrder(oid, true)
```

Comment:
I think we need to keep this so that the cancel matches are show on the order details page. (Make an order, settle 50% and cancel the remaining -> the order details page should show both of those matches.  We have special handling for the "cancel match" added in https://github.com/decred/dcrdex/pull/816)

![image](https://user-images.githubusercontent.com/6109680/98239041-f5ee7680-1f2c-11eb-9837-35cff4c11f78.png)

Reply:
I mean I guess we can get rid of the need for this special display if we all agree.  @buck54321 Referring to your fixes from https://github.com/decred/dcrdex/pull/816 being mostly restored in this PR.

Reply:
On second thought, excluding cancels in this method this would result in a breaking change on exported the `Core` API, and the `core.Order` generated by this method would be inconsistent with the `core.Order` generated by `(*trackedTrade).coreOrder`.

Let's just stick to the goal of fixing the filled amounts and percentages.

Reply:
changes reverted.